jhotdraw-gui/src/main/java/org/jhotdraw/gui/JDisclosureToolBarBeanInfo.java

```diff
@@ -150,45 +150,31 @@ public int getDefaultEventIndex() {
    * @return An image object representing the requested icon. May return null if no suitable icon is
    *     available.
    */
+  public java.awt.Image loadImage(String iconName, java.awt.Image icon) {
```

Comment:
Also, the comment above concerns the `getIcon` method, so it should be moved below it.